plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -113,6 +116,26 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
 
         is PrimitiveJavaType -> signatureForProjection(p.translateToKotlin())
     }
+
+    fun PageContentBuilder.DocumentableContentBuilder.funType(type: TypeConstructor) {
+        if (type.extension) {
+            signatureForProjection(type.projections.first())
+            text(".")
+        }
+
+        val args = if (type.extension) {
+            type.projections.drop(1)
+        } else
+            type.projections
```

Comment:
Please use brackets consistently. Either use them both before and after `else` or not use them at all

Reply:
Fixed.